pkgs/os-specific/linux/nvidia-x11/default.nix

```
@@ -128,14 +128,33 @@ rec {
   };
 
   # Last one supporting x86
-  legacy_390 = generic {
+  legacy_390 = let
+  # Source corresponding to https://aur.archlinux.org/packages/nvidia-390xx-dkms
```

```suggestion
    # Source corresponding to https://aur.archlinux.org/packages/nvidia-390xx-dkms
```

Lets not get bonked by the code formatter